pkg/storage/chunk/cache/cache.go

```diff
@@ -145,6 +146,18 @@ func New(cfg Config, reg prometheus.Registerer, logger log.Logger, cacheType sta
 		}
 	}
 
+	if cfg.LRUCache.Enabled {
+		cache, err := NewLRUCache(cfg.Prefix+"embedded-cache", cfg.LRUCache, reg, logger, cacheType)
```

Comment:
I assume "embedded cache" is used for the FIFO implementation.

```suggestion
		cache, err := NewLRUCache(cfg.Prefix+"inmemory-lru-cache", cfg.LRUCache, reg, logger, cacheType)
```